lib/pusher-redux.js

```diff
@@ -103,7 +106,7 @@ module.exports.subscribe = function (channelName, eventName, actionType) {
     var eventSubs = channelSubs[eventName];
     if (!eventSubs[actionType]) {
       eventSubs[actionType] = function (data) {
-        config.store.dispatch(pusherAction({ actionType: actionType, channelName: channelName, eventName: eventName, data: data }));
+        config.store.dispatch(pusherAction({ actionType: actionType, channelName: channelName, eventName: eventName, data: data, additionalParams }));
```

Comment:
this repo is not ES6 :-(
so should probably be `additionalParams: additionalParams`